pkg/build/builder/dockerutil.go

```diff
@@ -113,3 +118,60 @@ func buildImage(client DockerClient, dir string, dockerfilePath string, noCache
 	}
 	return client.BuildImage(opts)
 }
+
+// dockerRun mimics the 'docker run' command. It uses the Docker Remote API to
+// create and start a container and stream its logs to stdout and stderr. The
+// container is removed after it terminates.
+func dockerRun(client DockerClient, createOpts docker.CreateContainerOptions, logsOpts docker.LogsOptions) error {
+	// Create a new container.
+	glog.V(4).Infof("Creating container ...")
```

Comment:
It is in there to match with other `glog.V(4)` we have in the builder package.

Just printing createOpts with `%v` (or `%+v` or `%#v`) is not enough, since the interesting parts are pointers.

We'd need some custom printer or at least something like below:

```go
	glog.V(4).Infof("Creating container with options {Name:%q Config:%+v HostConfig:%+v} ...", createOpts.Name, createOpts.Config, createOpts.HostConfig)
```

That would produce output like this:

```
I0122 13:59:06.072966   17827 dockerutil.go:126] Creating container with options {Name:"" Config:&{Hostname: Domainname: User: Memory:0 MemorySwap:0 MemoryReservation:0 KernelMemory:0 CPUShares:0 CPUSet: AttachStdin:false AttachStdout:false AttachStderr:false PortSpecs:[] ExposedPorts:map[] StopSignal: Tty:false OpenStdin:false StdinOnce:false Env:[] Cmd:[
set -x
echo "Hello world!"
>&2 echo "this is in stderr"
cat /sys/fs/cgroup/cpu,cpuacct/cpu.shares
cat /sys/fs/cgroup/cpu,cpuacct/cpu.cfs_period_us
cat /sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us
cat /sys/fs/cgroup/memory/memory.limit_in_bytes
# exit 1
] DNS:[] Image:c91ab321f208 Volumes:map[] VolumeDriver: VolumesFrom: WorkingDir: MacAddress: Entrypoint:[sh -c] NetworkDisabled:false SecurityOpts:[] OnBuild:[] Mounts:[] Labels:map[]} HostConfig:&{Binds:[] CapAdd:[] CapDrop:[] GroupAdd:[] ContainerIDFile: LxcConf:[] Privileged:false PortBindings:map[] Links:[] PublishAllPorts:false DNS:[] DNSOptions:[] DNSSearch:[] ExtraHosts:[] VolumesFrom:[] NetworkMode: IpcMode: PidMode: UTSMode: RestartPolicy:{Name: MaximumRetryCount:0} Devices:[] LogConfig:{Type: Config:map[]} ReadonlyRootfs:false SecurityOpt:[] CgroupParent: Memory:0 MemorySwap:0 MemorySwappiness:0 OOMKillDisable:false CPUShares:1024 CPUSet: CPUSetCPUs: CPUSetMEMs: CPUQuota:-1 CPUPeriod:100000 BlkioWeight:0 Ulimits:[] VolumeDriver:}} ...
```

wdyt?